### PR TITLE
Bucket region caching and other minor changes.

### PR DESCRIPTION
**Pull request by benbart**

### CODE REVIEW

File: cloudformation/thin-egress-app.yaml [hunk 1]
```diff
@@ -101,7 +101,7 @@ Parameters:
 
   LambdaCodeS3Bucket:
     Type: String
-    Default: "asf.public.code"
+    Default: "asf.public.code.usw2"
```
Comment (bbuechler) [inline cloudformation/thin-egress-app.yaml:104]: I don’t think we want this. Public code should all be in one bucket, even if it’s not the “primary” region.

Comment (benbart) [reply cloudformation/thin-egress-app.yaml:104]: Isn't this where we're uploading our code to now?

Comment (bbuechler) [reply cloudformation/thin-egress-app.yaml:104]: Not that I’m aware of. I think there is a UW2 private code bucket, but public should stay in UE1

Comment (benbart) [reply cloudformation/thin-egress-app.yaml:104]: turns out it is:
https://jenkins.asf.alaska.edu/job/rain/job/thin-egress-app-pipeline-prod/79/console
`aws s3 cp ./tea-code-build.79.zip s3://asf.rain.code.usw2/thin-egress-app/`

Comment (bbuechler) [reply cloudformation/thin-egress-app.yaml:104]: asf.public.code Vs asf.rain.code

Comment (benbart) [reply cloudformation/thin-egress-app.yaml]: OK, it's gone

**Pull request merged**